src/api.ts

```diff
@@ -105,8 +105,10 @@ export class Api extends Connection implements ApiInterface {
                     planRefresh();
                 } catch (e) {
                     // remove tokens from localStorage if expired
-                    localStorage.removeItem('accessToken');
-                    localStorage.removeItem('refreshToken');
+                    if (e instanceof TokenExpiredError) {
```

Comment:
`refreshToken` isn't ws request, so here we can't catch `TokenExpiredError` because it relevant only for ws requests.

For now platform-back doesn't send error codes for tokenRefresh error, it need to be implemented in platform back. So current task can be done only after changes in platform back.